bool is_to_store
string stream_uri
---
bool success
string filename
string msg
uint32 duration_secs
uint32 wallclock_secs